navigator_engine/common/graph_visualizer.py

```diff
@@ -0,0 +1,209 @@
+import navigator_engine.model as model
+from dash import dcc
+import dash_cytoscape as cyto
+from dash import html
+from dash.dependencies import Input, Output
+
+
+graph_stylesheet = [  # Group selectors
+    {
+        'selector': 'node',
+        'style': {
+            'content': 'data(label)'
+        }
+    },
+    {
+        'selector': 'edge',
+        'style': {
+            # The default curve style does not work with certain arrows
+            'curve-style': 'bezier',
+            'target-arrow-shape': 'vee'
+        }
+    },
+
+    # Class selectors for nodes
+    {
+        'selector': '.red',
+        'style': {
+            'background-color': 'red',
+            'line-color': 'red'
+        }
+    },
+    {
+        'selector': '.blue',
+        'style': {
+            'background-color': 'blue',
+            'line-color': 'blue'
+        }
+    },
+    {
+        'selector': '.green',
+        'style': {
+            'background-color': 'green',
+            'line-color': 'green'
+        }
+    },
+    {
+        'selector': '.triangle',
+        'style': {
+            'shape': 'triangle'
+        }
+    },
+    {
+        'selector': '.square',
+        'style': {
+            'shape': 'square'
+        }
+    },
+
+    # Class selectors for edges
+    {
+        'selector': '.true',
+        'style': {'width': 5}
+    },
+    {
+        'selector': '.false',
+        'style': {'line-style': 'dashed'}
+    }
+]
+
+
+def create_visualizer(flask_app, dash_app) -> None:
+
+    dash_app.layout = html.Div(children=[
+        html.Div(
+            id='dropdown-container',
+            children=[
+                dcc.Dropdown(
+                    id='graph-dropdown',
+                    options=[],
+                    placeholder='Select a graph',
+                )
+            ]
+        ),
+        html.Hr(),
+        html.Div(
+            id='cytoscape-container',
+            children=[
+                cyto.Cytoscape(
+                    id='cytoscape-figure',
+                    layout={'name': 'preset'},
+                    style={'width': '100%', 'height': '400px'},
+                    elements=[
+                    ])
```

Comment:
I know it is picky, but I really do like all my brackets aligned with the same indentation.  Can you move the square bracket onto the line above and put the parenthesis in line with cyto.Cytoscape above?